src/features/views/components/ViewColumnDialog/choices.tsx

```diff
@@ -15,6 +15,7 @@ import PersonFieldConfig from './PersonFieldConfig';
 import PersonTagConfig from './PersonTagConfig';
 import SingleIconCardVisual from './SingleIconCardVisual';
 import SurveySubmitDateConfig from './SurveySubmitDateConfig';
+import theme from 'theme';
```

Comment:
This is actually not a great solution, because it assumes that the default export from `theme.ts` is always the correct theme to use. It's slightly better than hardcoding colors in _this_ file, but now it means that the colors are hardcoded in `theme.ts` instead.

In a future where we want to support multiple themes (e.g. a dark theme besides the default light theme) this still won't help us. That's why we want to use `useTheme()` (and equivalent) whenever possible.

All of this said, I think this is good enough for now and there is a lot of work that will need to be done to implement a dark theme anyway, so let's cross that bridge when we get there.

Reply:
aaa ok i understand.... merging as good enough for now!